src/tools/sdk/archive/index.md

```diff
@@ -3,9 +3,9 @@ title: "Index of Downloads"
 description: "Download specific stable and dev channel versions of the Dart SDK, Dartium, and the Dart API documentation."
 permalink: /install/archive
```

Comment:
Update the permalink to /tools/sdk/archive, and (wherever the breadcrumbs are defined) make the previous page be Dart SDK, not Install Dart.

Reply:
I've renamed this page to "Dart SDK Download", with a short title of just "Download", so that we get this heading w/ breadcrumbs:

> <img width="372" alt="screen shot 2018-04-02 at 15 48 41" src="https://user-images.githubusercontent.com/4140793/38212869-5a6afa92-368d-11e8-87fa-04b929cfcb23.png">

Reply:
We have to be careful about how we refer to these pages. This title might be good, but we don't want to imply that it's the preferred way to download the SDK.